android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/ddl/form/fields/DDLFieldRadioView.java

```diff
@@ -160,6 +160,15 @@ public void setParentView(View view) {
 
     @Override
     public void setUpdateMode(boolean enabled) {
+        List<Option> availableOptions = field.getAvailableOptions();
```

Comment:
You could try to write this method less java-like, taking advantages from the kotlin optional values and functional interfaces.